src/Icicle/Avalanche/ToJava.hs

```diff
@@ -280,10 +283,15 @@ primTypeOfPrim p
   dble     M.PrimDoubleExp  = Function "Math.exp"
   dble     M.PrimDoubleSqrt = Function "Math.sqrt"
 
-  cast      M.PrimCastIntOfDouble = Function "(int)"
-  cast      M.PrimCastDoubleOfInt = Function "(double)"
-  cast      M.PrimCastStringOfInt = Function "Integer.toString"
-  cast      M.PrimCastStringOfDouble = Function "Double.toString"
+  toInt    M.PrimToIntFloor      = Function "Math.floor"
+  toInt    M.PrimToIntCeiling    = Function "Math.ceil"
+  toInt    M.PrimToIntRound      = Function "Math.round"
+  toInt    M.PrimToIntTruncate   = Function "Math.trunc"
+
+  toDouble M.PrimToDoubleFromInt = Function "(double)"
+
+  toString M.PrimToStringFromInt    = Function "Integer.toString"
+  toString M.PrimToStringFromDouble = Function "Double.toString"
```

Comment:
We should probably just delete this, I don't think we're ever going to need it, thoughts @amosr  ?